hathor/indexes/tips_index.py

```diff
@@ -38,116 +38,26 @@ class TipsIndex(BaseIndex):
     TODO Use an interval tree stored in disk, possibly using a B-tree.
```

Comment:
Should we remove this TODO from this class?

Reply:
I think it's fine to leave it. The TODO proposes something different, which would completely replace the in-memory interval tree and make loading instantaneous. But realistically we will probably end up removing the tips-index when sync-v1 is removed before implementing this.